src/ngraph/runtime/cpu/builder/mlir_cpu_compiled_kernel.cpp

```diff
@@ -38,6 +38,17 @@ namespace ngraph
             {
                 auto& functors = external_function->get_functors();
 
+                CompiledKernel* compiled_kernel =
```

Comment:
I don't think fixing up the graph here is a good idea. Why not just leave it as-is and let compiler do the write look up to resolve the tensor ptr.